ui/components/multichain/network-list-menu/rpc-list-item.tsx

```diff
@@ -0,0 +1,93 @@
+import { RpcEndpointType } from '@metamask/network-controller';
+import React from 'react';
+import { infuraProjectId } from '../../../../shared/constants/network';
+import { Box, Text } from '../../component-library';
+import {
+  Display,
+  FlexDirection,
+  BorderStyle,
+  BorderColor,
+  TextColor,
+  TextVariant,
+  BackgroundColor,
+  BlockSize,
+} from '../../../helpers/constants/design-system';
+
+export const stripKeyFromInfuraUrl = (endpoint: string) => {
+  let modifiedEndpoint = endpoint;
+
+  if (modifiedEndpoint.endsWith('/v3/{infuraProjectId}')) {
+    modifiedEndpoint = modifiedEndpoint.replace('/v3/{infuraProjectId}', '');
+  } else if (modifiedEndpoint.endsWith(`/v3/${infuraProjectId}`)) {
+    modifiedEndpoint = modifiedEndpoint.replace(`/v3/${infuraProjectId}`, '');
+  }
```

Comment:
i think the first `endsWith()` is an incorrectly formed template string, but both these cases are the same anyway

Reply:
ohh... i understand now. nevermind